src/test/regress/specs/isolation_validate_vs_insert.spec

```diff
@@ -0,0 +1,39 @@
+#
+# How we organize this isolation test spec, is explained at README.md file in this directory.
+#
+
+# create distributed table to test behavior of VALIDATE in concurrent operations
```

Comment:
Minor: It'd be great to mention that `ALTER TABLE % VALIDATE CONSTRAINT` doesn't block inserts and selects on the table with this comment. 

Because, in general `ALTER TABLE` commands blocks all activity on the tables, and this one is unusual in that sense.